pkgs/development/libraries/libavif/default.nix

```diff
@@ -29,14 +36,17 @@ stdenv.mkDerivation rec {
     "-DAVIF_CODEC_DAV1D=ON" # best decoder (fast)
     "-DAVIF_CODEC_AOM_DECODE=OFF"
     "-DAVIF_BUILD_APPS=ON"
+    "-DAVIF_BUILD_GDK_PIXBUF=ON"
   ];
 
   nativeBuildInputs = [
     cmake
     pkg-config
+    makeWrapper
   ];
 
   buildInputs = [
+    gdk-pixbuf
```

Comment:
It will need to also go to `nativeBuildInputs` for the build time programs.

Reply:
Good catch... which makes me wonder, does `makeWrapper ${gdk-pixbuf}/bin/gdk-pixbuf-thumbnailer` pick the correct `gdk-pixbuf`?